.github/workflows/lighthouseci.yml

```
@@ -10,10 +10,10 @@ jobs:
   lighthouseci:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 19
       - run: npm install && npm install -g @lhci/cli@0.11.x
```

Thanks for the changes @leecalcote .This can be updated
```suggestion
      - run: npm install && npm install -g @lhci/cli@0.12.x
```

---

Oops, I missed your code suggestion. That looks good. Yes, let's include. I would have accepted the change into this PR, but didn't see it until after I hit the button. Perhaps, open a new PR...